docs/docs/results.md

```diff
@@ -156,3 +156,10 @@ job = Job.fetch(id='my_id', connection=redis)
 for result in job.results(): 
     print(result.created_at, result.type)
 ```
+
+To block until a result arrives, you can pass a timeout in seconds to `job.latest_result()`. If any results already exist, the latest result is returned immediately. If the timeout is reached without a result arriving a `None` object is returned.
```

Comment:
`If the timeout is reached without a result arriving, a `None` object is returned.` This sentence is missing a comma.